tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -145,7 +145,10 @@ private bool EnsureWebExperiencePack()
 
             Log.Logger()?.ReportInfo("DashboardView", $"{package.Id.FullName} Version: {major}.{minor}");
 
-            if ((major < stableVer) || (major == stableVer && minor < stableMin) || (major == stageVer && minor < stageMin))
+            // Min 400's version is 423.3800.0.0

```

Comment:
Nit: `if` statement with multiple intervals checks can easily produce silent bugs. My nit suggestion is we can break this statement into smaller components and use the [CompareTo](https://learn.microsoft.com/en-us/dotnet/api/system.version.compareto?view=net-7.0) to check if the target version is in range.

Something quick:
```cs
readonly Version minSupportedVersion400 = new Version(423, 3800);
readonly Version version500 = new Version(500, 0);
readonly Version minSupportedVersion500 = new Version(523, 3300);
readonly Version version600 = new Version(600, 0);

private bool IsVersionBetween(Version target, Version min, Version max) => target.CompareTo(min) >= 0 && target.CompareTo(max) < 0;
private bool IsVersionSupported(Version target) => IsVersionBetween(target, minSupportedVersion400, version500) || IsVersionBetween(target, minSupportedVersion500, version600);
```